### PR TITLE
First draft of audiences section

### PR DESCRIPTION
**Pull request by dmarti**
Related: #172

***
<a href="https://pr-preview.s3.amazonaws.com/w3ctag/privacy-principles/pull/193.html" title="Last updated on Nov 16, 2022, 11:46 PM UTC (20de140)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3ctag/privacy-principles/193/90459b8...20de140.html" title="Last updated on Nov 16, 2022, 11:46 PM UTC (20de140)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -426,6 +426,39 @@
...
+
+## Audience for this Document
+
+The primary audiences for this document are
+
+ * browser developers
+
+ * authors of W3C specifications
+
+ * reviewers of W3C specifications
```
Comment (torgo) [inline index.html:439]: Maybe we should be clear that it's applicable to all web specifications, not only ones developed at W3C.

File: index.html [hunk 2]
```diff
@@ -426,6 +426,39 @@
 
 </section>
 
+<section class="introductory">
+
+## Audience for this Document
+
+The primary audiences for this document are
+
+ * browser developers
```
Comment (darobin) [inline index.html:435]:
```suggestion
 * browser developers,
```

File: index.html [hunk 3]
```diff
@@ -426,6 +426,39 @@
...
+
+ * authors of W3C specifications
```
Comment (darobin) [inline index.html:437]:
```suggestion
 * authors of W3C specifications, and
```

File: index.html (hunk 1 above)
Comment (darobin) [inline index.html:439]:
```suggestion
 * reviewers of W3C specifications.
```

File: index.html [hunk 4]
```diff
@@ -426,6 +426,39 @@
...
+
+ * reviewers of W3C specifications
+
+Other audiences include
```
Comment (darobin) [inline index.html:441]:
```suggestion
Other audiences include:
```

File: index.html [hunk 5]
```diff
@@ -426,6 +426,39 @@
...
+
+ * web developers
```
Comment (darobin) [inline index.html:443]:
```suggestion
 * web developers,
```

File: index.html [hunk 6]
```diff
@@ -426,6 +426,39 @@
...
+
+ * policy makers
```
Comment (darobin) [inline index.html:445]:
```suggestion
 * policy makers, and
```

File: index.html [hunk 7]
```diff
@@ -426,6 +426,39 @@
...
+
+ * operators of privacy-related services
```
Comment (darobin) [inline index.html:447]:
```suggestion
 * operators of privacy-related services.
```

File: index.html [hunk 8]
```diff
@@ -426,6 +426,39 @@
...
+
+This document is intended to help its audiences address privacy concerns as early as possible in the life
+cycle of a new Web standard or feature.  Beginning with privacy in mind will help avoid the need to
```
Comment (darobin) [inline index.html:450]:
```suggestion
cycle of a new Web standard or feature, or in the development of Web products.  Beginning with privacy in mind will help avoid the need to 
```

**Review by darobin — CHANGES_REQUESTED** [review]
Most of it is nitpicks because I can't figure out multiline reviews.

File: index.html (hunk 1 above)
Comment (jyasskin) [inline index.html:439]: To reduce the extra whitespace:
```suggestion
 * browser developers
 * authors of W3C specifications
 * reviewers of W3C specifications
```

File: index.html (hunk 8 above)
Comment (jyasskin) [reply index.html:450]: Or just
```suggestion
cycle of a new feature.  Beginning with privacy in mind will help avoid the need to 
```

File: index.html (hunk 5 above)
[the remark this answers is not in the document]
Comment (dmarti) [reply index.html]: I will move web developers to primary audiences.

**Comment by dmarti** [comment]
All comments on this one fixed, at least 1 approving review is required to merge this pull request. Can merge then.  @darobin @jyasskin @torgo

**Review by jyasskin — APPROVED** [review]

**Pull request merged**